cocotb/utils.py

```diff
@@ -630,3 +614,134 @@ def extract_coro_stack(coro, limit=None):
     whitespace stripped; if the source is not available it is ``None``.
     """
     return traceback.StackSummary.extract(walk_coro_stack(coro), limit=limit)
+
+
+class _ActivehdlVersion(LooseVersion):
+    """
+    Version numbering class for Aldec Active-HDL.
+
+    NOTE: unsupported versions exist, e.g.
+    _ActivehdlVersion("10.5a.12.6914") > _ActivehdlVersion("10.5.216.6767")
+    """
+    pass
+
+
+class _CvcVersion(LooseVersion):
+    """
+    Version numbering class for Tachyon DA CVC.
+
+    Example:
+        >>> _CvcVersion("OSS_CVC_7.00b-x86_64-rhel6x of 07/07/14 (Linux-elf)") > _CvcVersion("OSS_CVC_7.00a-x86_64-rhel6x of 07/07/14 (Linux-elf)")
+        True
+    """
+    pass
```

Comment:
If we're declaring these not  (yet) for public consumption, then perhaps simpler to put them in a new file, `cocotb._sim_versions.ActivehdlVersion` - that is, make the module `_`-prefixed instead of the classes. That also leaves a good place to stick a docstring warning that these are not for public use.

Reply:
Done.